sdk/core/loader/boot.S

```diff
@@ -32,30 +32,30 @@ start:
 
 	la_abs			a3, bootStack
 	li				a1, BOOT_STACK_SIZE
-	cspecialr		ca2, mtdc
-	li				a4, ~CHERI_PERM_STORE_LOCAL
+	cspecialr		ca4, mtdc // Keep the RW memory root in ca4 throughout
+	li				a2, ~CHERI_PERM_STORE_LOCAL
```

Comment:
Sorry, what'd I get wrong about whitespace here?

Reply:
Admittedly looks like the whole file has the problem.

Reply:
Ah, I see.  OK, all the lines I've otherwise modified should have the correct whitespace, but I haven't fixed the file as whole to avoid making the PR any harder to review.